docs/development.adoc

```diff
@@ -448,6 +448,61 @@ in https://github.com/kadel/homebrew-odo[`kadel/homebrew-odo`].
 . Confirm the binaries are available in the GitHub release page.
 . Create a PR and update the file `build/VERSION` with the  latest version number.
 
+== Writing machine readable output code
+
+Here are some tips to consider when writing machine-readable output code.
+
+- Match similar Kubernetes / OpenShift API structures
+- Put as much information as possible within `Spec`
```

Comment:
`Put as much relevant information as possible within Spec as needed by the other teams` to prevent #2073